lib/BackgroundJobs/ClassifyFacesJob.php

```diff
@@ -43,6 +41,6 @@ protected function classify(array $files) : void {
 	 * @return int
 	 */
 	protected function getBatchSize() :int {
-		return $this->config->getAppValue('recognize', 'tensorflow.purejs', 'false') === 'false' ? self::BATCH_SIZE : self::BATCH_SIZE_PUREJS;
+		return intval($this->config->getAppValue('recognize', 'faces.batchSize', '100'));
```

Comment:
Maybe keep the default value as a constant?

Reply:
Moved to ClassifierJob